app/src/main/java/org/simple/clinic/setup/SetupActivity.kt

```diff
@@ -0,0 +1,44 @@
+package org.simple.clinic.setup
+
+import android.content.Context
+import android.os.Bundle
+import android.view.WindowManager
+import androidx.appcompat.app.AppCompatActivity
+import io.github.inflationx.viewpump.ViewPumpContextWrapper
+import org.simple.clinic.BuildConfig
+import org.simple.clinic.ClinicApp
+import org.simple.clinic.util.LocaleOverrideContextWrapper
+import java.util.Locale
+import javax.inject.Inject
+
+class SetupActivity : AppCompatActivity() {
+
+  companion object {
+    lateinit var component: SetupActivityComponent
```

Comment:
Hmmm... not really comfortable doing this :(

Reply:
I have made it a private property in a later commit. This was initially copied from the `TheActivity`.